cmip6_downscaling/data/cmip.py

```diff
@@ -269,3 +269,19 @@ def gcm_munge(ds: xr.Dataset) -> xr.Dataset:
         ds = ds.drop('height')
     ds = ds.squeeze(drop=True)
     return ds
+
+
+def get_gcm_grid_spec(gcm: str) -> str:
+    gcm_grid = load_cmip(
```

Comment:
this creates such a better string pattern than the ESMF regridder one! i went to look out of curiosity to see if there was anything we were missing and this one is way better. i wonder if we could contribute it as a more informative string-creation utility? https://github.com/pangeo-data/xESMF/blob/25bbb9a001960f46b31e5847c82216d54364ba07/xesmf/frontend.py#L356-L371